lib/Zonemaster/Backend/Config.pm

```diff
@@ -149,101 +144,103 @@ sub parse {
     if ( defined( my $value = $get_and_clear->( 'DB', 'database_host' ) ) ) {
         push @warnings, "Use of deprecated config property DB.database_host. Use MYSQL.host or POSTGRESQL.host instead.";
 
-        $obj->{_MYSQL_host} = $value
+        $obj->_set_MYSQL_host( $value )
           if $obj->DB_engine eq 'MySQL';
```

Comment:
What will happen if Mysql-host is already set by the new method, or reverse?

Reply:
I suggest, in a later PR, that it is not accepted to have both.

Reply:
You mean to only allow single assignment? It's surely possible but I'm not sure there's a way to do it that's worth its price in complexity. There's no shortage of little constraints in parse().